packages/bits/src/lib/image/image.component.ts

```diff
@@ -116,8 +126,9 @@ export class ImageComponent implements OnInit, AfterViewInit {
         let imageHtml: string = "";
         if (_has(image, "code") && _isString(image.code)) {
             imageHtml = image.code;
+            this.ariaLabel = image.name;
```

Comment:
It looks like `ariaLabel` will only be populated if `image` has a `code` property. What do you think about putting this outside of the if check and doing something like this:

```
this.ariaLabel = this.imageAlt || image.name;
```

That way we would default to using the `imageAlt` as the aria label and fall back to the `image.name` if no `imageAlt` is provided. It's just an idea.

Reply:
I had the same idea in the beginning but when I did it, screen readers read 2 times the same text. First, it reads from <img alt> and then from component aria-label. So I decided to left only alt. 
I'll add input ariaLabel to the component. So users will have possibility to set their own aria-label.